src/library_webgl.js

```diff
@@ -1424,7 +1428,7 @@ var LibraryGL = {
   },
 
   glCompressedTexImage2D: (target, level, internalFormat, width, height, border, imageSize, data) => {
-#if MAX_WEBGL_VERSION >= 2
+#if MAX_WEBGL_VERSION >= 2 && !MEMORY64
```

Comment:
Why this?

Reply:
I have not checked the newest versions of chrome/firefox, but up until recently calling webgl functions that directly accessed an array view via an offset did not work, when the array view was larger than 4GB. I will check, if this still holds true. If newer browser versions got rid of that limitation i can remove a lot of these checks.

Reply:
If I am not mistaken, in wasm32 when the array view is larger than **2GB**, there is the same problem.